lib/deploy_complexity/git.rb

```diff
@@ -7,7 +7,7 @@ module Git
 
     # converts origin/master -> master
     def safe_name(name)
-      name.chomp.sub(%r{^[^/]+/}, '')
+      name&.chomp&.sub(%r{^origin/}, '')
```

Comment:
are we really targeting Ruby 2.3 as `.ruby-version` says? We're using 2.5.7 on the monolith (based on looking at the same file there)

I ask because of a nitpick: `delete_prefix` was introduced in 2.5, and would maybe be clearer here. What do you think?

Reply:
Nice, +1 to using `delete_prefix`. I'll merge this and look into updating `.ruby-version` in a separate PR.